docs/xdr/usecases/playbook/ExtractIP_from_Url_country.md

```diff
@@ -0,0 +1,40 @@
+# Extract IP from url & Country  
+
+This use case describes how to extract an IP address from a URL and a country.  
+
+
+## Prerequisites
+
+- A SEKOIA.IO XDR licence
+- The user that configures the Playbook should have a role that contains:
+	* The SYMPHONY permissions
+- An API Key with a role that contains at least the following permission:
+	* `SIC_READ_ALERTS`
+
+> To create your API Key, follow this [documentation](../../../getting_started/generate_api_keys.md).
+
+## Playbook configuration
+
+Here is the playbook configuration to set:
+
+![Playbook ExtractIP](docs/assets/playbooks/library/UseCases/ExtractIP.png)
+
+Alert created => set module & trigger configuration
+Get Alert => **uuid** = `short_id` variables in **Alert created**
+
+
+### Extract IP from URL
+
+For example, we would like to extract the IP `65.74.70.888` from `url.original : /Service/Voice/SecuritySettings/111111/ManageIP/New/65.74.70.888`
+
+Read JSON File module,  **Jsonpath** = `{{ value.split()[0].split("/ManageIP/New/")[1] }}`
+
+   !!! note
+   You can find the CODE Feature section in documentation page and in particular built-in filters existing in Jinja [here](https://jinja.palletsprojects.com/en/3.0.x/templates/#builtin-filters). 
+   (split is coming from Python)
+
+### Extract IP from Country
```

Comment:
I don't see the link between the IP and the country? From what I understand, it again extracts an IP address from a URL 

And extract IP from country is not exact, it makes more sense to write "enrich an IP address with the country".

Reply:
I guess I have miss some information.

The use case is extracted from this [document](https://docs.google.com/document/d/1_D0eV-0bHYKxFkrS9pQ2CfDQmcxdzBJV/edit#) where it seems possible to the country of the IP address.

![image](https://user-images.githubusercontent.com/101662967/212119629-ec50fb97-9943-4caf-b3a8-19866d13d5b1.png)